src/Card/Card.tsx

```diff
@@ -61,7 +61,17 @@ export const Card: React.FC<CardProps> = ({
       paddingBottom: size === "large" ? 16 : 28,
     }}
   >
-    <div css={{ display: "flex", marginBottom: children ? 24 : 0 }}>
+    <div
+      css={{
+        display: "flex",
+        marginBottom:
+          !children ||
+          (Array.isArray(children) &&
+            children.filter((child) => !!child).length === 0)
+            ? 0
+            : 24,
```

Comment:
Could simplify with React.Children.toArray?

```suggestion
        marginBottom: React.Children.toArray(children).some(Boolean) ? 24 : 0
```

Reply:
Much better! Dont know how I ended up with this 😛